### PR TITLE
Wire-up excluded resource types list to the CRD check and update logging

### PR DESCRIPTION
**Pull request by ozevren**

### CODE REVIEW

**Comment by ozevren** [comment]
Unrelated failure in e2e-pilot-noauth-v1alpha3-v2
```
gateway "istio-ingressgateway" deleted
--- FAIL: TestIngressGateway503DuringRuleChange (62.84s)
	ingressgateway_test.go:263: Could not parse response codes from the client
=== RUN   TestVirtualServiceMergingAtGateway
```

File: galley/pkg/server/server.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Istio Authors
+// Copyright 2019 Istio Authors
```
Comment (ayj) [inline galley/pkg/server/server.go]: I believe copyright date corresponds to when the file was _created_ and not the latest update. cc @geeknoid

**Comment by ozevren** [comment]
This is the new GoLand version updating the copyright notice during commits. Given that they've added this as part of the commit model and enabled it by default, I assumed this was acceptable. Happy to revert if this is not the right thing to do.

Things LG otherwise?

**Comment by ayj** [comment]
The changes LGTM sans the updated copyright date.

**Comment by ozevren** [comment]
Updated Copyright dates.

**Comment by ozevren** [comment]
@duderino This wires-up the excludedResourceKinds to the CRD check. It ensures that Galley can start when CRDs are missing (and when explicitly set on the command-line).

**Pull request merged**